tests/bearlib/aspects/DecoratorsTest.py

```diff
@@ -46,22 +62,63 @@ def test_setting_priority(self):
             Setting('remove_unreachable_code', 'False'))
         self.section.append(
             Setting('minimum_clone_tokens', 40))
-        result = self.bear.execute()
-        self.assertEqual([False, 40], result)
+        result = self.list_to_dict(self.bear.execute())
+        self.assertEqual({'remove_unreachable_code': False,
+                          'minimum_clone_tokens': 40,
+                          'use_spaces': True}, result)
 
     def test_partial_mapping(self):
         self.section.aspects = AspectList([
             get_aspect('UnreachableCode')('py'),
         ])
-        result = self.bear.execute()
-        self.assertEqual([True, 10], result)
+        result = self.list_to_dict(self.bear.execute())
+        self.assertEqual({'remove_unreachable_code': True,
+                          'minimum_clone_tokens': 10,
+                          'use_spaces': True}, result)
 
     def test_no_mapping(self):
         self.section.aspects = AspectList([])
-        result = self.bear.execute()
-        self.assertEqual([False, 10], result)
+        result = self.list_to_dict(self.bear.execute())
+        self.assertEqual({'remove_unreachable_code': False,
+                          'minimum_clone_tokens': 10,
+                          'use_spaces': True}, result)
 
     def test_skip_mapping(self):
         self.section.aspects = None
-        result = self.bear.execute()
-        self.assertEqual([False, 10], result)
+        result = self.list_to_dict(self.bear.execute())
+        self.assertEqual({'remove_unreachable_code': False,
+                          'minimum_clone_tokens': 10,
+                          'use_spaces': True}, result)
+
+
+class MapAmbiguousSettingToAspectTest(unittest.TestCase):
+
+    def list_to_dict(self, tuple_list):
+        result_dict = {}
+        for pair in tuple_list:
+            result_dict[pair[0]] = pair[1]
+        return result_dict
+
+    def setUp(self):
+        self.section = Section('aspect section')
+        self.bear = RunDecoratedBear(self.section, None)
+
+    def test_mapping(self):
+        self.section.aspects = AspectList([
+            get_aspect('Indentation')('py', indent_type='tab'),
+        ])
+        result = self.list_to_dict(self.bear.execute())
+        self.assertEqual({'remove_unreachable_code': False,
+                          'minimum_clone_tokens': 10,
+                          'use_spaces': False}, result)
+
+    def test_setting_priority(self):
+        self.section.aspects = AspectList([
+            get_aspect('Indentation')('py', indent_type='tab'),
+        ])
+        self.section.append(
```

Comment:
why split this line?

Reply:
Sorry, I just saw that you asked the same on the previous review too. Will be even more careful next time. :)